project/src/main/java/com/google/sps/data/ProductSearchLibrary.java

```diff
@@ -234,8 +229,7 @@ public static void removeProductFromProductSet(
                 .setProductSearchParams(
                     ProductSearchParams.newBuilder()
                         .setProductSet(productSetPath)
-                        .addProductCategories(productCategory)
-                        .setFilter(filter))
```

Comment:
Why is the filter being removed?

Reply:
The filter is supposed to be used in the product search, and we really have no way of implementing the filters, so we removed them. We could have passed in an empty string for the filter, but that would mess with the values to be returned